src/extractor.js

```diff
@@ -1,6 +1,5 @@
 const extractComments = require('multilang-extract-comments');
 const jsYaml = require('js-yaml');
-
```

Comment:
Can you add this line break back in? Our coding standards wants it to be there:

![Screen Shot 2022-03-01 at 10 25 00 PM](https://user-images.githubusercontent.com/33762/156307190-5945ff55-122f-4c3e-a3b1-1ba66fa0fe6c.png)

(You can see these errors with `npm run lint`)